### PR TITLE
Solved violation open-closed principle && Strong coupling movement < …

### PR DESCRIPTION
**Pull request by Cas-B**
…bug,performance,maintainability

### CODE REVIEW

**Comment by rjbergTU** [comment]
No problems

File: src/main/java/org/group17/Units/Enemy.java [hunk 1]
```diff
@@ -48,8 +47,7 @@ public Enemy(Vector vLocation, GameMap gmMap) {
 	@Override
 	public void move(double dTimePerFrame) {
 		this.checkJump();
-		super.move(dTimePerFrame);
-		this.accelerate();
```
Comment (kusterlu) [inline src/main/java/org/group17/Units/Enemy.java]: it makes me very sad how you simply delete my hard sweaty desert work. i WANT to accelerate whenever i want to. <bitching,

Comment (Cas-B) [inline src/main/java/org/group17/Units/Enemy.java]: Your work is just replaced, your very hard sweaty work is still in use, no problem sweetheart <CaringPerson

**Comment by kusterlu** [comment]
my eclipse makes trouble when i try to solve merge. can you do it @Cas-B ?

**Comment by Cas-B** [comment]
What will break the tests as you didn't make in-code comments
I will of course solve the merge conflicts
The 0,00001 part is for making equals() check on differences above that decimal

**Comment by Cas-B** [comment]
Fixed the merge conflict.
If you can explain where you found broken tests as every tests says Okay at mine, please tell me and otherwise you can merge it:)

**Comment by kusterlu** [comment]
the actually are in-code comments, they are not coherently displayed.

**Comment by kusterlu** [comment]
ok, good to know. 
the conflics are solved and I will merge it.

**Pull request merged**